backend/src/server.py

```diff
@@ -401,6 +401,35 @@ async def python_info(_request: Request):
     return json({"python": sys.executable, "version": version})
 
 
+@dataclass
+class SystemStat:
+    label: str
+    percent: float
+
+
+@app.route("/system-stats", methods=["GET"])
+async def system_stats(_request: Request):
```

Comment:
Why did you call the end point "system stats"? Wouldn't "system usage" be better? 

When I first read the title of this PR, I thought that you moved the system info collection for error logs into the backend.